jme3-examples/src/main/java/jme3test/texture/TestSkyRotation.java

```diff
@@ -131,7 +131,7 @@ public void onAction(String name, boolean ongoing, float ignored) {
             System.out.print("rotate floor and sky leftward ...");
         } else if (name.equals("right")) {
             angle -= 0.1f; // radians
-            System.out.printf("rotate floor and sky spatials rightward ...");
+            System.out.print("rotate floor and sky spatials rightward ...");
```

Comment:
While before/after are equivalent, I wonder if the original writer meant to have these all glom onto one line or if they left out the linefeed.

Reply:
All system out prints in this class now have a linefeed

Reply:
Linefeeds reverted